### ISSUE TITLE
[issue] Server help messages print multiple times

### ISSUE THREAD

**Issue by bigangryrobot** [megaease/easegress#37]
Seeing a bit of duplication I think in the help message output for server and over-all this is way too overwhelming of a help page but that can be addressed elsewhere. See version here printed 2 times, as well as others

```
Usage of ./easegress-server:
      --api-addr string                               Address([host]:port) to listen on for administration traffic. (default "localhost:2381")
      --cluster-advertise-client-urls strings         List of this member’s client URLs to advertise to the rest of the cluster. (default [http://localhost:2379])
      --cluster-initial-advertise-peer-urls strings   List of this member’s peer URLs to advertise to the rest of the cluster. (default [http://localhost:2380])
      --cluster-join-urls strings                     List of URLs to join, when the first url is the same with any one of cluster-initial-advertise-peer-urls, it means to join itself, and this config will be treated empty.
      --cluster-listen-client-urls strings            List of URLs to listen on for cluster client traffic. (default [http://localhost:2379])
      --cluster-listen-peer-urls strings              List of URLs to listen on for cluster peer traffic. (default [http://localhost:2380])
      --cluster-name string                           Human-readable name for the new cluster, ignored while joining an existed cluster. (default "eg-cluster-default-name")
      --cluster-request-timeout string                Timeout to handle request in the cluster. (default "10s")
      --cluster-role string                           Cluster role for this member (reader, writer). (default "writer")
  -f, --config-file string                            Load server configuration from a file(yaml format), other command line flags will be ignored if specified.
      --cpu-profile-file string                       Path to the CPU profile file.
      --data-dir string                               Path to the data directory. (default "data")
      --debug                                         Flag to set lowest log level from INFO downgrade DEBUG.
      --force-new-cluster                             Force to create a new one-member cluster.
      --home-dir string                               Path to the home directory. (default "./")
      --labels stringToString                         The labels for the instance of Easegress. (default [])
      --log-dir string                                Path to the log directory. (default "log")
      --member-dir string                             Path to the member directory. (default "member")
      --memory-profile-file string                    Path to the memory profile file.
      --name string                                   Human-readable name for this member. (default "eg-default-name")
  -c, --print-config                                  Print the configuration.
  -v, --version                                       Print the version and exit.
      --wal-dir string                                Path to the WAL directory.
      --api-addr string                               Address([host]:port) to listen on for administration traffic. (default "localhost:2381")
      --cluster-advertise-client-urls strings         List of this member’s client URLs to advertise to the rest of the cluster. (default [http://localhost:2379])
      --cluster-initial-advertise-peer-urls strings   List of this member’s peer URLs to advertise to the rest of the cluster. (default [http://localhost:2380])
      --cluster-join-urls strings                     List of URLs to join, when the first url is the same with any one of cluster-initial-advertise-peer-urls, it means to join itself, and this config will be treated empty.
      --cluster-listen-client-urls strings            List of URLs to listen on for cluster client traffic. (default [http://localhost:2379])
      --cluster-listen-peer-urls strings              List of URLs to listen on for cluster peer traffic. (default [http://localhost:2380])
      --cluster-name string                           Human-readable name for the new cluster, ignored while joining an existed cluster. (default "eg-cluster-default-name")
      --cluster-request-timeout string                Timeout to handle request in the cluster. (default "10s")
      --cluster-role string                           Cluster role for this member (reader, writer). (default "writer")
  -f, --config-file string                            Load server configuration from a file(yaml format), other command line flags will be ignored if specified.
      --cpu-profile-file string                       Path to the CPU profile file.
      --data-dir string                               Path to the data directory. (default "data")
      --debug                                         Flag to set lowest log level from INFO downgrade DEBUG.
      --force-new-cluster                             Force to create a new one-member cluster.
      --home-dir string                               Path to the home directory. (default "./")
      --labels stringToString                         The labels for the instance of Easegress. (default [])
      --log-dir string                                Path to the log directory. (default "log")
      --member-dir string                             Path to the member directory. (default "member")
      --memory-profile-file string                    Path to the memory profile file.
      --name string                                   Human-readable name for this member. (default "eg-default-name")
  -c, --print-config                                  Print the configuration.
  -v, --version                                       Print the version and exit.
      --wal-dir string                                Path to the WAL directory.
```

**Comment by benja-wu** [comment]
Thanks for reporting, let me figure out what's wrong here.

**Comment by benja-wu** [comment]
Merged by #50
